frontend/src/services/graphql/Environment/listValidEnvironments.js

```diff
@@ -0,0 +1,24 @@
+import gql from 'graphql-tag';
```

Comment:
Minor Comment - could we change this import and for `listEnvironment` to be  `import { gql } from 'apollo-boost';` (similar to how we do for all other queries/mutations)

`apollo-boost` comes with `graphql-tag` package and want to keep imports consistent

Reply:
Strangely listEnvironment query also imported gql from `graphql-tag`. I just changed both in my last commit. There are no more imports using `graphql-tag`